<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockInterface</class>
 <widget class="QWidget" name="BlockInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlockInterface</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_20">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="pos_3">
      <attribute name="title">
       <string>POS</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QListWidget" name="posWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>25</number>
           </property>
           <item row="2" column="1">
            <widget class="QPushButton" name="selectBashPath">
             <property name="text">
              <string>影像Base路径</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="selectPosFile">
             <property name="text">
              <string>打开</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="clearPos">
             <property name="text">
              <string>清除pos文件</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="genBlock">
             <property name="text">
              <string>生成block文件</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="selectPath">
             <property name="text">
              <string>生成文件路径</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>选择要生成block的pos文件</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="imageBasePath"/>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEditTemplate"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="templateFile">
             <property name="text">
              <string>模板文件</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>150</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
